src/components/pages/Home.tsx

```diff
@@ -1,6 +1,6 @@
 'use client';
 
-import dynamic from 'next/dynamic';
+// ^ the use client directive should be removed as soon as possible once query-client provides a real ESm build
```

Comment:
Is there an issue open for that ?

Reply:
Yes see https://github.com/graasp/graasp-query-client/issues/626